cmd/minikube/cmd/config/addons_list.go

```diff
@@ -166,13 +166,30 @@ var printAddonsJSON = func(cc *config.ClusterConfig) {
 
 		addonBundle := assets.Addons[addonName]
 		enabled := addonBundle.IsEnabled(cc)
-
-		addonsMap[addonName] = map[string]interface{}{
-			"Status":  stringFromStatus(enabled),
-			"Profile": cc.Name,
+	        maintainer := addonBundle.Maintainer
+		docs := addonBundle.Docs
+		if addonPrintDocs{
+			addonsMap[addonName] = map[string]interface{}{
+				"Status":  stringFromStatus(enabled),
+				"Profile": cc.Name,
+				"Maintainer": maintainer,
+				"Docs":       docs,
+				
+			}
+			
+		}
+		if !addonPrintDocs{
+		        addonsMap[addonName] = map[string]interface{}{
+				"Status":  stringFromStatus(enabled),
+				"Profile": cc.Name,
+		
+		        }
```

Comment:
```suggestion
		addonsMap[addonName] = map[string]interface{}{
                        "Status":  stringFromStatus(enabled),
                        "Profile": cc.Name,
                }
                if addonPrintDocs {
                        addonsMap[addonName]["Maintainer"] = addonBundle.Maintainer
                        addonsMap[addonName]["Docs"] = addonBundle.Docs
```
This code can be refactored to greatly reduce duplication and lines of code